### PR TITLE
Decouple Page and App.Model, promote Home to a proper page

### PR DESCRIPTION
**Pull request by kerams**
Attempt to address #209.

### CODE REVIEW

File: src/Client/App.fs [hunk 1]
```diff
@@ -143,36 +135,15 @@ open Client.Style
...
-        | _ -> [ ]
-
-    | Page.WishList ->
-        match model.SubModel with
-        | WishListModel m ->
-            [ div [ ] [ WishList.view m (WishListMsg >> dispatch) ]]
-        | _ -> [ ]
+    match model.PageModel with
+    | HomePageModel ->
+        Home.view ()
```
Comment (kerams) [inline src/Client/App.fs]: `site.css` edits needed in order to surround this in a `div` like the other cases,

**Comment by forki** [comment]
Yes that looks much better

File: src/Client/App.fs (hunk 1 above)
Comment (forki) [reply src/Client/App.fs]: Maybe the div can be moved out

File: src/Client/App.fs [hunk 2]
```diff
@@ -143,36 +135,15 @@ open Client.Style
...
+
+    | LoginModel m ->
+        [ div [] [ Login.view m (LoginMsg >> dispatch) ] ]
+
+    | WishListModel m ->
+        [ div [] [ WishList.view m (WishListMsg >> dispatch) ] ]
```
Comment (kerams) [inline src/Client/App.fs]: Do you mean these divs?

Comment (forki) [reply src/Client/App.fs]: yes

**Comment by kerams** [comment]
Let me know if you'd like to merge it as is and I'll update the tomato example.

**Comment by forki** [comment]
yes please fix readme and I'll merge. Thank you!

**Comment by forki** [comment]
thanks!

**Pull request merged**